File: cakk-batch/src/main/java/com/cakk/batch/config/ComponentScanConfig.java
package com.cakk.batch.config;

import org.springframework.context.annotation.ComponentScan;
import org.springframework.context.annotation.Configuration;

import com.cakk.infrastructure.persistence.config.DataSourceConfig;
import com.cakk.infrastructure.persistence.config.JpaConfig;

@Configuration
@ComponentScan(basePackages = {
	"com.cakk.infrastructure.cache",
	"com.cakk.batch"
}, basePackageClasses = {
	DataSourceConfig.class,
	JpaConfig.class
})
public class ComponentScanConfig {
}

File: cakk-admin/src/main/kotlin/com/cakk/admin/config/ComponentScanConfig.kt
package com.cakk.admin.config

import org.springframework.context.annotation.ComponentScan
import org.springframework.context.annotation.Configuration

@Configuration
@ComponentScan(basePackages = [
	"com.cakk.domain",
	"com.cakk.admin"
])
class ComponentScanConfig
